language/evm/examples/sources/Table.move

```diff
@@ -0,0 +1,35 @@
+/// This module defines Table for EVM.
+module Evm::Table {
```

Comment:
Maybe there was some confusion around naming things, but these are just maps, not tables. Tables are a new scalable storage type that will sit alongside the current resource system. 

Mind just renaming this to `Map` :D 

Looks good otherwise though!

Reply:
This is supposed to be scalable storage type thing. For example, they aren't enumerable. Also the scalable storage thing we need for EVM Move contracts. Perhaps let's review this API from this viewpoint again? Nothing is set in stone right now regards how it is defined.